src/coverlet.core/Enums/LogLevel.cs

```diff
@@ -1,4 +1,4 @@
-namespace Coverlet.Console.Logging
+namespace coverlet.core.Enums
```

Comment:
Why?

Reply:
This was related to the changes on ```ILogger```. See below.